AdguardHome/README.md

```diff
@@ -40,3 +40,21 @@
 1. Browse to 10.0.5.3:3000 and follow the setup wizard
 1. Update your DNS Servers to 10.0.5.3 (or your custom ip) in all your DHCP configs.
 1. Access the AdguardHome like you would normally.
+
+## Troubleshooting
+
+If you get the following error:
+
+```
+Error adding network: failed to create macvlan: cannot assign requested address
+```
+
+When starting the container then the MAC address you generated is not good. You can cheat at this point and look at the address of `br$VLAN.mac` with `ifconfig br$VLAN.mac` and use that value.
+
+To start over you must remove the container and the macvlan device:
+
+```
+podman container rm adguardhome
+podman network rm dns -f # expect an error here
+ip link delete br$VLAN.mac
+```
```

Comment:
Would suggest adding a final note (even if obvious) of which step in the process to restart at, just to pre-empt any future questions.

Reply:
Sure makes sense.